pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -4380,14 +4319,13 @@ func TestIsRunning(t *testing.T) {
 		},
 		want: false,
 	}, {
-		name: "taskrun failed: retries remaining",
+		name: "taskrun failed: retried",
 		rpt: ResolvedPipelineTask{
 			PipelineTask: &v1beta1.PipelineTask{Name: "task", Retries: 1},
-			TaskRun:      makeFailed(trs[0]),
+			TaskRun:      withRetries(makeFailed(trs[0])),
 		},
-		want: true,
+		want: false,
 	}, {
-
 		name: "run failed: retries remaining",
```

Comment:
Doesn't have to be in this PR but I think this case name shall be not be `failed`?

Reply:
Ah, then what would be the name you'd like to suggest?

Reply:
Could be wrong but it doesn't feel like a 'failed' here? (As I am comparing with others that have `makeFailed(trs[0])` are actually failed ones?

Reply:
I see, yeah your thought makes sense. But for Custom Task Beta, it is failed as long as its status is marked as failed.  any more thoughts?